nemo/collections/asr/parts/submodules/conformer_modules.py

```diff
@@ -89,6 +90,16 @@ def __init__(
                 pos_bias_v=pos_bias_v,
                 max_cache_len=MHA_max_cache_len,
             )
+        elif self_attention_model == 'rel_pos_local_attn':
```

Comment:
How about using the longformer instead of local attention in the names as we already had local attention feature? This one is an sufficient implementation of that which is an adaptation of LongFormer for relative positional encoding. What do you think?

Reply:
For now I called this local_attn because it's missing global attention tokens from longformer. I was thinking if we add that later it would be more correct to call that option longformer